indico/web/templates/layout/dialog_base.html

```diff
@@ -0,0 +1,24 @@
+{% block page %}
+    <div class="{% block dialog_class %}dialog-page{% endblock %}"
+        {% if self.dialog_id()|trim -%}
```

Comment:
I would say that
``` jinja
<div class="{% block dialog_class %}dialog-page{% endblock %}"
     {% if self.dialog_id()|trim -%}
         id="{% filter trim %}{% block dialog_id %}{% endblock %}{% endfilter %}"
     {%- endif -%}
>
    {% if self.description() or self.info() %}
        <header>
``` 
looks better than
``` jinja
<div class="{% block dialog_class %}dialog-page{% endblock %}"
     {% if self.dialog_id()|trim -%}
         id="{% filter trim %}{% block dialog_id %}{% endblock %}{% endfilter %}"
     {%- endif %}>
    {% if self.description() or self.info() %}
        <header>
```
because in the first case you can clearly see that the `if` belongs to the data attributes (hence the weird indentation) whereas in the second example it feels like a indentation bug since it differs only by 1 space from the next `if` statement.

Opinions @OmeGak @pferreir ?